### PR TITLE
add segment_elytra script

### PR DESCRIPTION
**Pull request by kwadraterry**
Added a script for elythra segmentation from individual beetle images

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,3 +56,6 @@ python3 individual_beetles.py --images <path to group_images> --csv <path to met
```
Comment (egrace479) [inline README.md:58]:
```suggestion
To crop out elytra from the individual images, run:
```

File: segmentation/segment_elythra.py [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+        
+        #save mask with cv2 to preserve pixel categories
+        print(f"Mask path:{output_path}")
+        save_anns(masks, image, output_path)
+
+    return
+
+
+if __name__ == "__main__":
+    main()
```
Comment (egrace479) [inline segmentation/segment_elythra.py:128]:
```suggestion
    main()

```

**Review by egrace479 — APPROVED** [review]

File: segmentation/segment_elythra.py
Comment (egrace479) [inline segmentation/segment_elythra.py:1]: This file should be renamed to `segment_elytra.py`.

File: README.md [hunk 3]
```diff
@@ -55,4 +55,7 @@ python3 individual_beetles.py --images <path to group_images> --csv <path to met
 **FYI**: The script to crop out individual beetles works well for the images that have coords_pix_length and coords_pix_width information correctly align to beetles. However, there are a couple images where this is not the case, and thus the segmentation of beetles will not result in a nice crop of the individual beetles.
 
 
-
+To crop out elytra from the individual images, run:
+```
+python3 segment_elythra.py --images <path to images> --result <path to folder where results will be saved>
```
Comment (egrace479) [inline README.md:60]:
```suggestion
python3 segment_elytra.py --images <path to images> --result <path to folder where results will be saved>
```

**Review by egrace479 — COMMENTED** [review]
Just noticed the filename typo.

**Review by ramirezmichelle — APPROVED** [review]

File: segmentation/segment_elythra.py
Comment (ramirezmichelle) [reply segmentation/segment_elythra.py:1]: @kwadraterry could you rename the file with the correct spelling and then I think we are good to merge! 😊

**Comment by hlapp** [comment]
@egrace479 or @ramirezmichelle anything missing still before this can be merged?

**Comment by egrace479** [comment]
> @egrace479 or @ramirezmichelle anything missing still before this can be merged?

The file rename and associated change to instructions noted above ([here](https://github.com/Imageomics/2018-NEON-beetles-processing/pull/5#discussion_r1722076647) and [here](https://github.com/Imageomics/2018-NEON-beetles-processing/pull/5#discussion_r1722076783))

**Comment by hlapp** [comment]
I was going to suggest to just go ahead and do it, so we can wrap this up.

**Comment by kwadraterry** [comment]
Renamed the files, sorry I didn't see it earlier

File: segmentation/segment_elytra.py [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+        
+        #save mask with cv2 to preserve pixel categories
+        print(f"Mask path:{output_path}")
+        save_anns(masks, image, output_path)
+
+    return
+
+
+if __name__ == "__main__":
+    main()
```
Comment (egrace479) [inline segmentation/segment_elytra.py:128]:
```suggestion
    main()

```

**Pull request merged**